app/controllers/organizations_controller.rb

```diff
@@ -31,6 +31,9 @@ def create
 
     respond_to do |format|
       if @organization.save
+        if params[:organization][:attachment]
```

Comment:
you can use `organization_params` if it's defined, smth like this `organization_params[:attachment]`